src/main/java/com/egov/icops_integrationkerala/enrichment/IcopsEnrichment.java

```diff
@@ -106,8 +106,10 @@ private AdditionalFields convertProcessReportData(IcopsProcessReport icopsProces
         AdditionalFields additionalFields = new AdditionalFields();
         log.info("IcopsProcessReport : {}", icopsProcessReport);
         List<Field> fieldsList = new ArrayList<>();
+        RequestInfo requestInfo = new RequestInfo();
+        String processUniqueId = idgenUtil.getIdList(requestInfo,config.getEgovStateTenantId(),config.getIdName(),null,1).get(0);
```

Comment:
not here. here we are reading response from icops. need to add id gen when sending request to icops